docs/grid/custom_filter.md

```diff
@@ -4,6 +4,124 @@ Sylius Grids come with built-in filters, but there are use-cases where you need
 
 To add a new filter, we need to create an appropriate class and form type.
 
+## Using attributes
+
+{% code title="src/Grid/Filter/SuppliersStatisticsFilter.php" lineNumbers="true" %}
```

Comment:
I think it should be in singular form, same as at most parts of the sylius.

```suggestion
{% code title="src/Grid/Filter/SupplierStatisticsFilter.php" lineNumbers="true" %}
```

Reply:
This is a copy of the previous documentation, if we change this we should change the "legacy way" documentation too.